src/include/sof/math/sqrt_int16.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Could the file name be just sqrt.h? If there are later other word length versions of function they could be added to same file.